manager-bundle/skeleton/config/config.yaml

```diff
@@ -111,6 +120,7 @@ doctrine:
     orm:
         auto_mapping: true
         auto_generate_proxy_classes: false
+        enable_lazy_ghost_objects: true
```

Comment:
This is recommended, however, there is an open issue regarding a performance degradation: https://github.com/doctrine/orm/issues/11087

Reply:
Yes but it's good for feature completion and also, not setting it is deprecated because Doctrine will apparently drop their own proxy stuff. So let's keep this as is and follow the issue.